### PR TITLE
fix: Log `rttvar` in qlog

### PR DESCRIPTION
**Pull request by larseggert**

### CODE REVIEW

File: neqo-transport/src/qlog.rs [hunk 1]
```diff
@@ -340,7 +340,7 @@ pub fn metrics_updated(qlog: &NeqoQlog, updated_metrics: &[QlogMetric]) {
                 QlogMetric::MinRtt(v) => min_rtt = Some(v.as_secs_f32() * 1000.0),
                 QlogMetric::SmoothedRtt(v) => smoothed_rtt = Some(v.as_secs_f32() * 1000.0),
                 QlogMetric::LatestRtt(v) => latest_rtt = Some(v.as_secs_f32() * 1000.0),
-                QlogMetric::RttVariance(v) => rtt_variance = Some(*v as f32),
+                QlogMetric::RttVariance(v) => rtt_variance = Some(v.as_secs_f32() * 1000.0),
```
Comment (mxinden) [inline neqo-transport/src/qlog.rs:343]:
```suggestion
                QlogMetric::RttVariance(v) => rtt_variance = Some(v.as_millis_f32()),
```

**Review by mxinden — APPROVED** [review]
Otherwise looks good to me.

>     ; all following rtt fields are expressed in ms

https://quicwg.org/qlog/draft-ietf-quic-qlog-quic-events.html#name-metrics_updated

Slightly related note, why is qlog not using SI units?

File: neqo-transport/src/qlog.rs (hunk 1 above)
Comment (mxinden) [reply neqo-transport/src/qlog.rs:343]: Ah, sorry, I missed the nightly-only flag.

**Review by mxinden — APPROVED** [review]

**Comment by larseggert** [comment]
Are there other qlog probes we are missing?

**Comment by mxinden** [comment]
I assume most missing ones are tracked in https://github.com/mozilla/neqo/issues/528 @larseggert.

**Pull request merged**